organizations/serializers.py

```diff
@@ -11,17 +16,47 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = Organization
         fields = '__all__'
+    
+    def validate_website(self, value):
+        # website URL has to be unique, empty string is not unique
+        if value == '':
+            return None
+        return value
+    
+    def validate_linkedin(self, value):
+        # linkedin URL has to be unique, empty string is not unique
+        if value == '':
+            return None
+        return value
+    
+    def validate_twitter(self, value):
+        # twitter URL has to be unique, empty string is not unique
+        if value == '':
+            return None
+        return value
+    
+    def validate_facebook(self, value):
+        # facebook URL has to be unique, empty string is not unique
+        if value == '':
+            return None
+        return value
+    
+    def validate_instagram(self, value):
+        # instagram URL has to be unique, empty string is not unique
+        if value == '':
+            return None
+        return value
```

Comment:
I am fairly sure that empty strings fail the URL validator check, is this not the case?

Reply:
Yea empty string would fail the validator. I think I changed them because when I was testing using the django rest framework web UI, it sends empty strings when I didn't input any link so my requests were failing. So I extended the validator to change empty strings to None instead of failing. They are probably unnecessary, I will remove them.

Reply:
if you tested it and it passes the check then i think its better to keep it like this, that way we dont have a bunch of empty strings